src/servers/http_server_v2.cc

```diff
@@ -1812,12 +1813,13 @@ HTTPAPIServerV2::EVBufferToInput(
       const char* shm_region = nullptr;
       if (CheckSharedMemoryData(
               request_input, &shm_region, &offset, &byte_size)) {
-        if (request_input.FindMember("data") == request_input.MemberEnd()) {
-          return TRTSERVER_ErrorNew(
-              TRTSERVER_ERROR_INVALID_ARG,
-              "must not specify 'data' field in request input when using "
-              "shared memory");
-        }
+        // Uncomment after client fix (Don't send data in case of shared memory)
```

Comment:
Shouldn't the client fix be included in this PR?

Reply:
Tanmay is making some fixes to the client. This will be a part of those fixes.

Reply:
I think the fix is merged #1278